android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java

```diff
@@ -763,6 +765,7 @@ public void onMeetingStatusChanged(MeetingStatus meetingStatus, int errorCode, i
     updateVideoView();
 
     sendEvent("MeetingEvent", getMeetErrorName(errorCode), meetingStatus);
+    sendEvent("MeetingStatus", meetingStatus.name());
```

Comment:
Is `meetingStatus.name()` returning the same as `meetingStatus`?

In general, `MeetingEvent` and `MeetingStatus` look the same to me for both iOS and android so I am not sure if adding another type is really needed.

It looks to me that in `src/events.ts` you could still achieve the same when listening to existing `MeetingEvent`.

Reply:
Not true.
In this case you can see order of arguments.

In MeetingEvent you receive events like this: 
`{ event: "success", status: "MEETING_STATUS_CONNECTING" }`.
Most other events works different: `{ event: "screenShareStopped" }`

In MeetingStatus you receive: 
`{ event: "MEETING_STATUS_CONNECTING" }`


Main goal was to fix format without breaking existing apps.
Also it provides grouping by specific events, so we can control it better, giving typings, etc.

I don't think sending few channels can be problem.

Reply:
Right, I understand the structure of the event is different.
But my point is that older event returns extra param that you can simply filter out on TypeScript side - i.e. in TypeScript you can just use `MeetingEvent` `status` to be your new `event`.

Reply:
It is not usual to have random parameter for event.
If you send event, so it would be great to receive `event`.
It could be better if all events works this way, but it works different.

I was testing event few days ago and it was very confusing when I receive events in random parameter. Sometimes: `event`, sometimes `status`.

So I wanted to fix it, before giving any typings.
Actually I wanted to give typings for all events, but it was more complicated than I expected.